Add Maximizer to DeFi Tokenlist

0xMadxi opened the pull request:
Adding MAXI token information for the Maximizer project (https://maxi.xyz).

Token asset has already been merged, reference: https://github.com/pangolindex/tokens/pull/117

Thank you as always~

bmino commented:
Hey, the minor version should be incremented to `5` with the new accompanying timestamp, and this looks good to go

0xMadxi commented:
> Hey, the minor version should be incremented to `5` with the new accompanying timestamp, and this looks good to go

rebased and updated to new `minor` version

bmino commented:
thanks!
